ui/src/learningReflections/js/learningReflections.js

```diff
@@ -14,31 +14,32 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
     // Provides the Learning Reflections framing to the Storytelling Tool
     fluid.defaults("sjrk.storyTelling.learningReflections", {
         gradeNames: ["sjrk.storyTelling.page"],
+        distributeOptions: [{
+            target: "{that > menu > templateManager}.options.templateConfig.templatePath",
+            record: "%resourcePrefix/src/learningReflections/templates/learningReflections-menu.handlebars"
+        },
+        {
```

Comment:
Two distributeOptions blocks at Line 21-28 are for configuring the same option of a same component, I feel combining them into one makes it easier to read:
```
{
    record: {
        messagesPath: "%resourcePrefix/src/learningReflections/messages/learningReflectionMessages.json",
        templatePath: "%resourcePrefix/src/learningReflections/templates/learningReflections-masthead.handlebars"
    }
    target: "{that > masthead > templateManager}.options.templateConfig",
}
```

The same comment applies to line 29-36 and other places. Thanks.